posthog/dags/common/resources.py

```diff
@@ -79,3 +83,22 @@ def create_resource(self, context: dagster.InitResourceContext) -> psycopg2.exte
             password=self.password,
             cursor_factory=psycopg2.extras.RealDictCursor,
         )
+
+
+class PostHogAnalyticsResource(dagster.ConfigurableResource):
+    personal_api_key: str | None
```

Comment:
This might work out of the box, but you might need to configure secrets for both US/EU if those are not configured for Dagster on the charts. I think the reason it works for web pods it is because there is something configured for them already. 
 https://github.com/PostHog/charts/blob/c396db92cb6d6d9e28ff5eeba6dd9bfe5f360137/argocd/dagster/values/prod-eu.yaml

In another words, I am not sure we're inheriting those values: https://github.com/PostHog/charts/blob/b150e5704d521586eec20e2aa6957392553a5c03/argocd/posthog/values/common.yaml#L190 or if we need to do something else for Dagster